packages/volto/locales/ca/LC_MESSAGES/volto.po

```diff
@@ -4652,10 +4652,10 @@ msgstr ""
 msgid "daterangeFacet"
 msgstr ""
 
-#. Default: "delete"
+#. Default: "delete {type} block"
 #: components/manage/Blocks/Block/EditBlockWrapper
-msgid "delete"
-msgstr "Suprimeix"
```

Comment:
Please keep the existing translations. Even if they need to be updated, that is better than having no translation.